iOS/Models/Contact.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Contact.swift
+//  Jibber
+//
+//  Created by Benji Dodgson on 9/28/21.
+//  Copyright © 2021 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+import Contacts
+import PhoneNumberKit
+
+struct Contact: Avatar, Hashable {
```

Comment:
Used to show what contacts have already been invited. UI not implemented yet to show this.